### PR TITLE
squirreldisk: Add version 0.3.4

### LINKED ISSUE #10630: [Request]: SquirrelDisk
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

SquirrelDisk

### Description

Cross-platform disk usage analysis tool

### Homepage

https://squirreldisk.com/

### Download Link(s)

https://github.com/adileo/squirreldisk/releases

### Some Indication of Popularity/Repute

The GitHub repository for SquirrelDisk has 159 stars at the time of writing, as well as a listing on [AlternativeTo](https://alternativeto.net/software/squirreldisk/about). It has received generally favorable reviews and is cross-platform, so users may already be familiar with it from other OSes.

### PR DESCRIPTION
**Pull request by Zliced13**
Closes #10630

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/squirreldisk.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "version": "0.3.4",
+    "homepage": "https://www.squirreldisk.com/",
+    "description": "A beautiful, super fast disk usage analysis tool. Built with rust.",
+    "license": "AGPL-3.0",
+    "notes": [
+        "This app depends on MSEdgeWebview2 and VB-Cable Virtual Audio Device to function properly.",
```
Comment (sidewinder94) [inline bucket/squirreldisk.json:7]: A dependency to VB-Cable ? I can't find any reference to that on their repository.

**Review by sidewinder94 — COMMENTED** [review]
Nitpick over the notes.

**Comment by Zliced13** [comment]
Sorry that was a typo.

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**